addons/stock_account/security/ir.model.access.csv

```diff
@@ -6,4 +6,5 @@ access_stock_picking_invoicing_payments,stock.picking,stock.model_stock_picking,
 access_stock_move_invoicing_payments_readonly,stock.move,model_stock_move,account.group_account_readonly,1,0,0,0
 access_stock_move_invoicing_payments,stock.move,model_stock_move,account.group_account_invoice,1,1,1,0
 access_stock_valuation_layer,access_stock_valuation_layer,model_stock_valuation_layer,stock.group_stock_manager,1,1,1,0
-access_stock_valuation_layer_revaluation,access_stock_valuation_layer_revaluation,model_stock_valuation_layer_revaluation,stock.group_stock_manager,1,1,1,0
\ No newline at end of file
+access_stock_valuation_layer_revaluation,access_stock_valuation_layer_revaluation,model_stock_valuation_layer_revaluation,stock.group_stock_manager,1,1,1,0
```

Comment:
Useless change in line 9

Reply:
There was no end-line, it is automatically added when I save, that is the change, should I remove it?